packages/fx-core/src/plugins/resource/bot/errors.ts

```diff
@@ -112,12 +108,14 @@ export class AADAppCheckingError extends PluginError {
 export class CreateAADAppError extends PluginError {
   constructor(innerError?: any) {
     super(ErrorType.User, CreateAppError.name, CreateAppError.message(), [], innerError);
+    InferInnerError(this, GraphErrorCodes);
```

Comment:
Maybe we could use "resolveInnerError"? the function name would be better starting with lower case

Reply:
`resolveInnerError` is okay for me, update it soon.